portalnet/src/storage.rs

```diff
@@ -419,7 +422,7 @@ impl PortalStorage {
         // Delete furthest data until our data usage is less than capacity.
         while self.capacity_reached()? {
             let id_to_remove =
-                // Always expect a content id if capacity_reached(), even at 0kb capacity
+                // Always expect a content id if capacity_reached(), even at 0mb capacity
```

Comment:
I realized when I read this comment that I don't understand: why won't it be None with a 0mb capacity?

After digging a bit, I think it was a more helpful comment before the commit: 201dba58

Now, I'd say it's more accurate to have a comment like:
```suggestion
                // An empty database cannot have .capacity_reached() be true, because it's only true when content size > capacity,
                // so content ID will always be Some() inside the loop.
```